### DISCUSSION TITLE
Kategorie oblíbených

### DISCUSSION THREAD

**Discussion by rblaha15** [jaro-jaro/DPMCB#131] [Q&A] [answered]
Přijde mi zvláštní, že když změním datum, tak v oblíbených se nic nezmění. To ale dává smysl, protože je tam napsáno `jede DNES`. Jak by to tedy bylo správně?

**Comment by rblaha15** [comment]
Cc @jaro-jaro, @Tlakova-mycka

**Comment by jaro-jaro** [comment]
asi "Jede vybraný den" a měnilo se to

**Reply by rblaha15** [reply]
Nebo by se tam dosadil. Např. `Jede 4. 4. 2023`, nebo `Jede 17. 8. 2023`

**Comment by jaro-jaro** [comment]
nebo by tam byla jako první kategorie "Jede vybraný den", pokud by nebyl vybrán dnešek, tak by pod tím bylo jede dnes, kde by byly všechny oblíbené, které jedou dnes včetně tich, které jedou vybrané datum a pod tím ještě jede jindy, kde by byly ty busy, které nebyly ani v jedné z předchozích kategoriích

**Reply by rblaha15** [reply]
Tohle se mi líbí nejvíc, ale bojím se, aby to nebylo zbytečně moc komplikovaný

**Reply by jaro-jaro** [reply]
jo, to by mohlo

**Comment by rblaha15** [comment, marked as answer]
Kategorie `dnes` bude předělána na vybraný datum. Takto to bude 1) jednoduché 2) konzistentí přes celou aplikaci
